### ISSUE TITLE
Unlock Dialog:  Crash #2 (onBadPassword)

### ISSUE THREAD

**Issue by HashEngineering** [HashEngineering/dash-wallet#112]
15 users, 16 crashes on all Android versions.
```
java.lang.IllegalStateException: 
  at android.app.Fragment.getResources (Fragment.java:814)
  at android.app.Fragment.getText (Fragment.java:826)
  at de.schildbach.wallet.ui.UnlockWalletDialogFragment$1.onBadPassword (UnlockWalletDialogFragment.java:67)
  at de.schildbach.wallet.ui.CheckWalletPasswordTask$1$2.run (CheckWalletPasswordTask.java:51)
  at android.os.Handler.handleCallback (Handler.java:751)
  at android.os.Handler.dispatchMessage (Handler.java:95)
  at android.os.Looper.loop (Looper.java:154)
  at android.app.ActivityThread.main (ActivityThread.java:6776)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run (ZygoteInit.java:1518)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1408)
```

**Comment by HashEngineering** [comment]
@sambarboza could this be fixed by #109 ?

**Comment by HashEngineering** [comment]
Fixed with f54a88115a8ea7ac683d47b4072eac7f83a4843b.
